src/platform/datapath_raw_xdp_win.c

```diff
@@ -1065,7 +1056,11 @@ CxPlatDpRawInitialize(
                 Status =
                     CxPlatDpRawInterfaceInitialize(
                         Xdp, Interface, ClientRecvContextLength);
-                if (QUIC_FAILED(Status)) {
+                if (Status == QUIC_STATUS_NOT_FOUND && CxPlatListIsEmpty(&Xdp->Interfaces)) {
```

Comment:
This isn't the same as `HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND)`. We could define a `QUIC_STATUS_FILE_NOT_FOUND` and use that.